packages/tpetra/CMakeLists.txt

```diff
@@ -602,60 +602,118 @@ LIST (APPEND ${PACKAGE_NAME}_ETI_LORDS "int")
+      ENDIF ()
+      GLOBAL_SET(HAVE_TPETRA_INST_INT_LONG_LONG_DEFAULT ${Teuchos_ENABLE_LONG_LONG_INT})
+    ELSE ()
+      GLOBAL_SET(HAVE_TPETRA_INST_INT_LONG_LONG_DEFAULT ON)
+    ENDIF ()
 
-IF (DEFINED Tpetra_INST_INT_LONG) # if the user set it explicitly (either ON or OFF)
-  IF (Tpetra_INST_INT_LONG) # if the user set it to ON explicitly
-    # If the user explicitly set Tpetra_INST_INT_LONG=ON, then turn
-    # off GO = long long by default.  We have to go through the
-    # two-level IF statement because the Tpetra_INST_INT_LONG option
-    # doesn't exist yet; we have not yet encountered the
-    # TRIBITS_ADD_OPTION_AND_DEFINE statement (below) that defines it
-    # and gives it a default value.  (Note the circular dependency
-    # between the default value of Tpetra_INST_INT_LONG and
-    # Tpetra_INST_INT_LONG_LONG.  This is deliberate.  The point is
-    # that only one of them is defined by default at a time.
-    GLOBAL_SET(HAVE_TPETRA_INST_INT_LONG_LONG_DEFAULT OFF)
-  ENDIF ()
-ENDIF ()
+    IF (DEFINED Tpetra_INST_INT_LONG) # if the user set it explicitly (either ON or OFF)
+      IF (Tpetra_INST_INT_LONG) # if the user set it to ON explicitly
+        # If the user explicitly set Tpetra_INST_INT_LONG=ON, then turn
+        # off GO = long long by default.  We have to go through the
+        # two-level IF statement because the Tpetra_INST_INT_LONG option
+        # doesn't exist yet; we have not yet encountered the
+        # TRIBITS_ADD_OPTION_AND_DEFINE statement (below) that defines it
+        # and gives it a default value.  (Note the circular dependency
+        # between the default value of Tpetra_INST_INT_LONG and
+        # Tpetra_INST_INT_LONG_LONG.  This is deliberate.  The point is
+        # that only one of them is defined by default at a time.
+        GLOBAL_SET(HAVE_TPETRA_INST_INT_LONG_LONG_DEFAULT OFF)
+      ENDIF ()
+    ENDIF ()
 
-# If GO = long long is OFF by default, turn ON GO = long by default.
-# If GO = long long is ON by default, turn OFF GO = long by default.
-# The point is to have EXACTLY one potentially 64-bit GO type enabled
-# by default.  I say "potentially" because the C++ standard only
-# guarantees long to be at least 32 bits.
-IF (HAVE_TPETRA_INST_INT_LONG_LONG_DEFAULT)
-  GLOBAL_SET(HAVE_TPETRA_INST_INT_LONG_DEFAULT OFF)
-ELSE ()
-  GLOBAL_SET(HAVE_TPETRA_INST_INT_LONG_DEFAULT ON)
-ENDIF ()
+    # If GO = long long is OFF by default, turn ON GO = long by default.
+    # If GO = long long is ON by default, turn OFF GO = long by default.
+    # The point is to have EXACTLY one potentially 64-bit GO type enabled
+    # by default.  I say "potentially" because the C++ standard only
+    # guarantees long to be at least 32 bits.
+    IF (HAVE_TPETRA_INST_INT_LONG_LONG_DEFAULT)
+      GLOBAL_SET(HAVE_TPETRA_INST_INT_LONG_DEFAULT OFF)
+    ELSE ()
+      GLOBAL_SET(HAVE_TPETRA_INST_INT_LONG_DEFAULT ON)
+    ENDIF ()
+
+    # GO = unsigned is NOT enabled by default.
```

Comment:
It doesn't really matter I think since the `IF enable deprecated ELSE disable deprecated ENDIF` will basically get removed here in a few months anyhow when we delete the deprecated things.